workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+
+
+#### Group By
+
+The Group By field is the outer most aggregation made in your query. Metric aggregations are contingent on the initial metric selected for your widget. Group your initial metric with resource attributes, metric attributes, or another metric based on the selected metric. 
+
+The below example tracks CPU utilization by host; it is grouping the `system.cpu.utilization` metric by all unique instances of the `host.name` metric:
+
+
+![](https://lh7-us.googleusercontent.com/ecBBt0x3KYh_2QyUksFy16HIKqlpl3k4PUyxIDjgrHD6M1iNkOiS8iR0P7vkHqktOCj8IdhZf-N8o6falplfl8KtNlHE4mFZFr65O_4ibg0OjXrxR3lL5zSPzLhKWppc_2YzQ6T_6ukb8VXkdA4EaYU)
+
+The following table displays all available methods for aggregating your chosen metrics:
+
+| __Aggregation Type__ | __Description__                                                |
+| :------------------- | :------------------------------------------------------------ |
+| __Average By__       | Average of all values in selected time range for given metric |
+| __Sum By__           | Sum of all values in selected time range for given metric |
+| __Minimum By__       | Minimum of all values in selected time range for given metric |
+| __Maximum By__       | Maximum of all values in selected time range for given metric |
+| __Any By__           | Gathers all possible values in selected time range for given metric __and/or__ attribute |
+| __Unique By__        | Count of unique values in selected time range for given metric __and/or__ attribute |
+| __Count By__         | Count of all values in selected time range for given metric __and/or__ attribute |
+| __Aggregated By__    | Groups all values in selected timerange for given metric __and/or__ attribute |
+
+#### Aggregate By
+
+The Aggregate By field is a nested aggregation technique that allows you to aggregate your initial query, the inner layer of data. The below example is grouping the `container.memory.percent` metric by averaging the values of the `container.name` attributes and summing `container.memory.percent`:
+
+![](https://lh7-us.googleusercontent.com/ohE5zbuYTvxfE_A3tAdR-ccxs11a_Z8XBPEmcDOLn4-jEe7pFGIwtOQDPkm4OwTH6h1-hmEUGZpFltgzQ1i2CwOn8gugkjRmIC4GM_YcFqGmnrCh1PBtHlFteYKpwiDSXCz_EZMfckZp-wbqhV-pWx4)
+
+#### Filters
+
+The filter feature is used to remove unwanted metrics within your query. The list of available filters is automatically available by the metadata pulled from all metrics being queried in the dashboard. The below example is filtering out any `host.id` containing `ip-172-31-13-218.us-east-2.compute.internal`:
+
+![](https://lh7-us.googleusercontent.com/_50CgdVxjgsqeqS6U3S8Rud9llopVXSi97d2WXCo4shRYJp79HjUs8gGTq_ZLihDDlP4wpi3IqCvaT6z4NpnhXAQWzhNWbe2uUS89Gw4Nd8ww_RX95Ce5p4LsR_SBO3-0LSp3sAht3heL01QDKgCAQQ)
+
+The following table displays all available filter functions for removing unwanted metrics:
```

Comment:
Can remove "for removing unwanted metrics" since we already called that out in the filter intro